openesdh-repo/src/test/java/dk/openesdh/repo/services/files/OeFilesServiceImplIT.java

```diff
@@ -82,21 +133,32 @@ public void setUp() {
         long timestamp = new Date().getTime();
         testComment = "Test Comment On file " + timestamp;
         file1 = filesService.addFile(owner1, "test_file_" + timestamp + ".txt", MimetypeMap.MIMETYPE_TEXT_PLAIN, fileBytes, testComment);
+        caseRef = caseDocTestHelper.createCaseBehaviourOn(TEST_FILES_CASE, folderService.getCasesRootNodeRef(),
+                AuthenticationUtil.getAdminUserName());
+
+        NodeRef adminRef = personService.getPerson(AuthenticationUtil.getAdminUserName());
+        testFileTitle = "test_file_" + timestamp + ".txt";
+        testFileRef = filesService.addFile(adminRef, testFileTitle,
```

Comment:
Maybe you could use file1 in test? another similar file creation is not necessary for all other tests. also I would move case creation and deletion to this particular test where it is used